src/DependencyInjection/HttplugExtension.php

```diff
@@ -419,6 +419,7 @@ function ($id) {
             ->addArgument([
                 'client_name' => $clientName,
             ])
+            ->addTag('httplug.client')
```

Comment:
i'd make this a public constant so that things using the tag can use it explicitly.

Reply:
Adjusted! 

I'll update the documentation as well.